diff --git a/browser/base/content/browser-context.inc.xhtml b/browser/base/content/browser-context.inc.xhtml
--- a/browser/base/content/browser-context.inc.xhtml
+++ b/browser/base/content/browser-context.inc.xhtml
@@ -367,6 +367,10 @@
                 data-l10n-id="main-context-menu-add-engine"/>
       <menuitem id="context-searchselect"/>
       <menuitem id="context-searchselect-private"/>
+      <menuitem id="context-zen-translate-page"
+                label="Перевести на русский"
+                hidden="true"
+                disabled="true"/>
       <menuitem id="context-translate-selection"
                 data-l10n-id="main-context-menu-translate-selection"/>
       <menu id="context-ask-chat"/>
