diff --git a/toolkit/modules/moz.build b/toolkit/modules/moz.build
index 784fcfecb6d8e542c8e9f9f89d41517e4075ea52..29f1c565e35a777492180aaacaf737e8952f91ae 100644
--- a/toolkit/modules/moz.build
+++ b/toolkit/modules/moz.build
@@ -216,6 +216,8 @@ EXTRA_JS_MODULES += [
     "URLKeywordStopWords.sys.mjs",
     "UrlUtils.sys.mjs",
     "WebChannel.sys.mjs",
+    "ZenPiPGestureMotion.sys.mjs",
+    "ZenPiPTrackpad.sys.mjs",
 ]
 
 MOZ_SRC_FILES += [
@@ -305,6 +307,7 @@ for var in (
     "DLL_SUFFIX",
     "DEBUG_JS_MODULES",
     "OMNIJAR_NAME",
+    "ZEN_FIREFOX_VERSION"
 ):
     DEFINES[var] = CONFIG[var] or ""
 
