diff --git a/widget/cocoa/nsCocoaWindow.h b/widget/cocoa/nsCocoaWindow.h
--- a/widget/cocoa/nsCocoaWindow.h
+++ b/widget/cocoa/nsCocoaWindow.h
@@ -318,6 +318,10 @@
   void DispatchAPZWheelInputEvent(mozilla::InputData& aEvent);
   nsEventStatus DispatchAPZInputEvent(mozilla::InputData& aEvent);
 
+  enum class PiPTrackpadPhase { Start, End, Cancel };
+  MOZ_CAN_RUN_SCRIPT_BOUNDARY void DispatchPiPTrackpadPhase(
+      PiPTrackpadPhase aPhase);
+
   void DispatchDoubleTapGesture(mozilla::TimeStamp aEventTimeStamp,
                                 LayoutDeviceIntPoint aScreenPosition,
                                 mozilla::Modifiers aModifiers);
@@ -462,6 +466,10 @@
   void SetHideTitlebarSeparator(bool) override;
   bool IsMacTitlebarDirectionRTL() override;
   void SetCustomTitlebar(bool) override;
+  void SetWindowClass(const nsAString& aXulWinType,
+                      const nsAString& aXulWinClass,
+                      const nsAString& aXulWinName) override;
+  void UpdateWindowMaterial();
   void UpdateThemeGeometries(
       const nsTArray<ThemeGeometry>& aThemeGeometries) override;
   void LockAspectRatio(bool aShouldLock) override;
@@ -675,6 +683,7 @@
   // show (e.g. a window created by detaching a tab from a fullscreen window).
   bool mIsInitialFullscreenSuppressed = false;
   bool mWasShown = false;
+  bool mIsBrowserWindow = false;
 
   int32_t mNumModalDescendants = 0;
 
